src/command.c

```diff
@@ -11196,23 +11196,25 @@ int GetUserHelp (char *helpTkn)
         MrBayesPrint ("                    are assuming. The default is 'strict', which corresponds to  \n");
         MrBayesPrint ("                    the standard clock model where the evolutionary rate is      \n");
         MrBayesPrint ("                    constant throughout the tree. For relaxed clock models, you  \n");
-        MrBayesPrint ("                    can use 'cpp', 'tk02', 'igr', or 'iln'.                      \n");
+        MrBayesPrint ("                    can use 'cpp', 'tk02', 'wn', 'igr', or 'iln'.                \n");
         MrBayesPrint ("                    'cpp' invokes a relaxed clock model where the rate evolves   \n");
         MrBayesPrint ("                    according to a Compound Poisson Process (CPP) (Huelsenbeck   \n");
         MrBayesPrint ("                    et al., 2000).                                               \n");
-        MrBayesPrint ("                    'tk02' invokes the Brownian Motion model described by Thorne \n");
-        MrBayesPrint ("                    and Kishino (2002). [autocorrelated lognormal distributions] \n");
-        MrBayesPrint ("                    'igr' invokes the Independent Gamma Rate (IGR) model where   \n");
-        MrBayesPrint ("                    each branch has an independent rate drawn from a gamma       \n");
-        MrBayesPrint ("                    distribution (LePage et al., 2007).                          \n");
+        MrBayesPrint ("                    'tk02' invokes the geometric Brownian Motion model described \n");
+        MrBayesPrint ("                    by Thorne and Kishino (2002). [autocorrelated lognormal]     \n");
+        MrBayesPrint ("                    'wn' invokes the white noise model (LePage et al., 2007)     \n");
+        MrBayesPrint ("                    where each branch has an independent rate drawn from a gamma \n");
+        MrBayesPrint ("                    distribution with variance propotional to the branch length. \n");
+        MrBayesPrint ("                    'igr' invokes the Independent Gamma Rate model. The differ-  \n");
+        MrBayesPrint ("                    ence from 'wn' is that these gamma distributions are i.i.d.  \n");
+        MrBayesPrint ("                    thus the variances do not depend on the branch lengths.      \n");
+        MrBayesPrint ("                    'iln' invokes the Independent Lognormal model. It is similar \n");
+        MrBayesPrint ("                    with 'igr' but the rates are i.i.d. lognormal distributions.\n");
         MrBayesPrint ("                    Each of the relaxed clock models has additional parameters   \n");
         MrBayesPrint ("                    with priors. For the CPP model, it is 'cppratepr' and        \n");
         MrBayesPrint ("                    'cppmultdevpr'; for the TK02 model, it is 'tk02varpr'; for   \n");
         MrBayesPrint ("                    the IGR  model, it is 'igrvarpr'.                            \n");
```

Comment:
Replace this line with "the WN model, it is 'wnvarpr'; for the IGR model, it is 'igrvarpr'; for the ILN model, it is 'ilnvarpr'.